tests/test_transformers.py

```diff
@@ -763,3 +764,118 @@ def test_binarize_works_in_gridsearch(self, train_iris_dataset):
         model = Model(grid)
         result = model.score_estimator(train_iris_dataset)
         assert isinstance(result, Result)
+
+
+class TestRareFeatureEncoder(TransformerBase):
+    def test_rare_feature_encoder_returns_correctly_series(self):
+        threshold = 2
```

Comment:
Don't need to declare them here - just use them directly in the RareFeatureEncoder instance